### PR TITLE
APIGOV-22429 Create Manage product plans page

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@georgiana-caras Please review and let me know when this functionality will be released (at which time this PR will be merged and the "Functionality coming soon" note in the Product management page will be deleted). Also, this page overlaps some of the content in https://docs.axway.com/bundle/amplify-central/page/docs/manage_product_foundry/foundry_product_management/index.html.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Objectives
+
+Learn how to create and configure the product plan using the Product Foundry WebUI.
+
+## Concepts
+
+**Plan** - contains the details about how to charge consumers, how often and how much.
+
+**Quotas** - describes the itemized units per resource or group of resources in product, and how much of those units they are entitled to use over a period.
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:17]: billing period

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+**Units** - defines the billable units. For example: Transactions, Seats, Messages. Used with the quotas to describe the pricing and how many units a consumer is entitled to use.
+
+## Product plan states
+
+| State      | Description                                                                                                   |
+|------------|---------------------------------------------------------------------------------------------------------------|
+| Draft      | Indicates that the plan is not listed in the Marketplace. Changes can be made to the plan.                |
+| Active     | Indicates that the plan is listed in the Marketplace. Allows new subscriptions.                               |
+| Deprecated | Indicates that the plan is listed in the marketplace, but new subscriptions are not allowed. Plan is not visible on the plan page.                                                                                                            |
+| Archived   | All associated subscriptions to this plan will be expired and the plan will be removed from the Marketplace.  |
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:28]: Archived: After a plan is deprecated with an auto-expiration date, the plan will be automatically to archived the movement the last subscription moves to expired.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+**Units** - defines the billable units. For example: Transactions, Seats, Messages. Used with the quotas to describe the pricing and how many units a consumer is entitled to use.
+
+## Product plan states
+
+| State      | Description                                                                                                   |
+|------------|---------------------------------------------------------------------------------------------------------------|
+| Draft      | Indicates that the plan is not listed in the Marketplace. Changes can be made to the plan.                |
+| Active     | Indicates that the plan is listed in the Marketplace. Allows new subscriptions.                               |
+| Deprecated | Indicates that the plan is listed in the marketplace, but new subscriptions are not allowed. Plan is not visible on the plan page.                                                                                                            |
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:27]: Deprecated: Indicates that new subscriptions on this plan are not allowed. Consumers with an existing subscription can continue to stay on this plan until their subscription expires.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Comment by lbadenhop** [comment]
@georgiana-caras Your changes have been added. Please review.

**Review by cbordier — COMMENTED** [review]
I approve the changes

**Pull request merged**